src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannBase.cs

```diff
@@ -334,7 +334,7 @@ protected string HandleJsonSelector(SelectorBlock selector, JToken parentObj, Di
                     throw new NotSupportedException();
                 }
 
-                if (setting.Type != "password" && indexerLogging)
+                if (setting.Type != "password" && setting.Name != "apikey" && setting.Name != "rsskey" && indexerLogging)
```

Comment:
@Qstick does it make sense to not log `setting.Type == "info"` as well since that isn't really useful?